src/audio/drc/Testbench_drc_lin2dB/Results/file.txt

```diff
@@ -0,0 +1,640 @@
+        idx  testvector Fixlog10linear 
```

Comment:
are these just test results? if they are not used in the code we should not commit it.

Reply:
Yes, these are test results, I will remove it